tools/run-integration-tests.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+set -ex
+
+[[ -d ${0%/*} ]] && cd "${0%/*}"/../
+
+fedora_version=40
+if [[ -n $1 ]]; then
+  fedora_version=$1
+fi
```

Comment:
`fedora_version=${1:-40}`?

Reply:
Applied to new version, good suggestion!